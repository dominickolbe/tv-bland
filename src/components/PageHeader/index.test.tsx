import { shallow } from "enzyme";
import React from "react";
import { PageHeader } from "./";

describe("<PageHeader />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(<PageHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
