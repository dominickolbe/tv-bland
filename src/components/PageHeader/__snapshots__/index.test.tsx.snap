// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> renders snapshots 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-root-1"
  maxWidth="lg"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={8}
      xs={12}
    >
      <WithStyles(ForwardRef(Typography))
        component="div"
        variant="h2"
      >
        <Styled(MuiBox)
          fontWeight="fontWeightBold"
        >
          TV Bland
        </Styled(MuiBox)>
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Container))>
`;
